references/detection/transforms.py

```diff
@@ -293,11 +293,13 @@ def __init__(
         target_size: Tuple[int, int],
         scale_range: Tuple[float, float] = (0.1, 2.0),
         interpolation: InterpolationMode = InterpolationMode.BILINEAR,
+        antialias=True,
```

Comment:
Had to add antialias support because it'd be False otherwise by default for tensors. There's no BC requirements so we could just hard-code `antialias=True` below in the calls to `resize()` instead of adding a parameter here, but it doesn't change much. LMK what you prefer.

Reply:
IDC. Unless there is some other opinion, let's keep it the way it is.